pkg/vault/contracts/Router.sol

```diff
@@ -672,9 +673,9 @@ contract Router is IRouter, RouterCommon, ReentrancyGuardTransient {
      * @param amountUnderlyingRaw Amount of underlying tokens that will be deposited into the buffer
      * @param amountWrappedRaw Amount of wrapped tokens that will be deposited into the buffer
      * @param sharesOwner Address of contract that will own the deposited liquidity. Only
-     *        this contract will be able to remove liquidity from the buffer
+     * this contract will be able to remove liquidity from the buffer
      * @return issuedShares the amount of tokens sharesOwner has in the buffer, expressed in underlying token amounts
-     *         (it is the BPT of vault's internal linear pools)
+     * (it is the BPT of vault's internal linear pools)
```

Comment:
```suggestion
     * (it is the BPT of vault's ERC4626 buffers)
```

I believe you fixed this in another PR, but I'm leaving the comment here in case that fix was in a different file.